docs/fastdds/xml_configuration/making_xml_profiles.rst

```diff
@@ -78,6 +78,100 @@ Please, refer to :ref:`xml_profiles` for further information regarding loading p
 
     To load dynamic types from XML files see the :ref:`Usage` subsection of :ref:`xmldynamictypes`.
 
+.. _loading_raw_xml_profiles:
+
+Get QoS from raw XML profiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Although the :ref:`standard procedure <loadingapplyingprofiles>` is to first load XML profiles, and then create entities
+given the profile name of choice, it is also possible to get the desired QoS from the XML profile directly so it can be
+modified before being used.
+For this purpose, the following public methods are available:
+
+.. collapse:: DomainParticipantFactory
```

Comment:
Don't know how to present this information in a clear and fitting manner. These are two alternative versions none of which I really like.